dashboard/config/routes.rb

```diff
@@ -344,6 +344,8 @@ module OPS
     get 'workshop_dashboard/*path', to: 'workshop_dashboard#index'
     get 'workshop_dashboard', to: 'workshop_dashboard#index'
 
+    get 'teacher_application', to: 'teacher_application#teacher_application'
```

Comment:
Like above, why not `teacher_application#new`? This is more standard. Also now I realize the `new` controller action above is never executed, since this route actually points the the `teacher_application` action, which directly renders the view of the same name since there is no matching controller method and no controller logic (such as cancan authorization) to stop it.